cwms-data-api/src/main/java/cwms/cda/api/TextTimeSeriesController.java

```diff
@@ -0,0 +1,357 @@
+import java.util.logging.Logger;
+import javax.servlet.http.HttpServletResponse;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.jooq.DSLContext;
+
+
+public class TextTimeSeriesController implements CrudHandler {
+    private static final Logger logger = Logger.getLogger(TextTimeSeriesController.class.getName());
+    private static final String TAG = "Text-TimeSeries";
+    public static final String TS_ID = "ts-id";
+    public static final String REPLACE_ALL = "replace-all";
+    private final MetricRegistry metrics;
+
+
+
+    public TextTimeSeriesController(MetricRegistry metrics) {
+        this.metrics = metrics;
+    }
+
+    @NotNull
+    protected TimeSeriesTextDao getDao(DSLContext dsl) {
+        return new TimeSeriesTextDao(dsl);
+    }
+
+
+    private Timer.Context markAndTime(String subject) {
+        return Controllers.markAndTime(metrics, getClass().getName(), subject);
+    }
+
+
+    @OpenApi(
+            queryParams = {
+                    @OpenApiParam(name = OFFICE, required = true, description = "Specifies the owning office of "
+                            + "the Text TimeSeries whose data is to be included in the response."),
+                    @OpenApiParam(name = TS_ID, required = true, description = "Specifies the ts-id of the "
+                            + "text timeseries"),
+                    @OpenApiParam(name = TIMEZONE,  description = "Specifies "
+                            + "the time zone of the values of the begin and end fields (unless "
+                            + "otherwise specified). If this field is not specified, "
+                            + "the default time zone of UTC shall be used."),
+                    @OpenApiParam(name = START, required = true, description = "The start of the time window"),
+                    @OpenApiParam(name = END, description = "The end of the time window. If specified the text associated with all times from start to end (inclusive) is deleted."),
+                    @OpenApiParam(name = VERSION_DATE, description = "The version date for the time series.  If not specified, the minimum or maximum version date (depending on p_max_version) is used."),
+                    @OpenApiParam(name = MAX_VERSION, type = Boolean.class, description = "Whether to use the maximum version date if p_version_date is not specified."),
+                    @OpenApiParam(name = Controllers.MIN_ATTRIBUTE, type = Long.class, description = "The minimum attribute value to delete. If not specified, no minimum value is used."),
+                    @OpenApiParam(name = Controllers.MAX_ATTRIBUTE, type = Long.class, description = "The maximum attribute value to delete. If not specified, no maximum value is used."),
+                    @OpenApiParam(name = "mode", required = true, type = TimeSeriesTextMode.class,
+                            description = "Type of Text TimeSeries to retrieve. Options are:\n"
```

Comment:
<img width="646" alt="image" src="https://github.com/USACE/cwms-data-api/assets/47677856/68be3a6e-3471-4d53-a2ce-a805f2739425">

formatting on the Mode renders funky